src/edu/csus/ecs/pc2/core/PermissionGroup.java

```diff
@@ -142,7 +142,6 @@ private void initialize() {
         feederPermissionList.addPermission(Type.TEST_RUN);
         feederPermissionList.addPermission(Type.VIEW_CLARIFICATIONS);
         feederPermissionList.addPermission(Type.VIEW_RUNS);
```

Comment:
I am a little concerned about the ANSWER_CLARIFICAITON permission.  Will this preclude the processing of answered clarifications received on the event feed?  This is one of the things we are adding for WF - the ability to read and process submitted clarifications AND the judge's responses to those clarifications received on the event feed.  Removing this permission may cause that not to work.  Based on my review of the code (PacketHandler), I think it may NOT work.

This also brings up another question in InternalController.java where this same permission is used (but in a different context).  There is a routine called InternalController.canCheckoutRunsAndClars().  The routine checks two permissions: JUDGE_RUN || ANSWER_CLARIFICATION.  Note the || (or).  The routine checks one OR the other, not AND.  It turns out the only place this is used is in the same module and it does appear that OR is the desired operation, but the routine appears to be misnamed.  Should an issue be created for this, or just a simple CI along with this PR, since it was noticed while reviewing this PR?

Reply:
Yes I also suspect that the answer clar from the EF for the FEEDER would fail if the ANSWER_CLARIFICAITON
is not present.   For that reason alone this is no longer a One Approval PR, because at this time it requires discussion
to firm up the requirements.   The real concern is that a person using shadow could answer a clar... and an answer
might be delayed on the Primary CCS.

re: renaming the canCheckoutRunsAndClars, note that it is an English vs computer speak naming and also
is combining two different permissions.   As such another issue could be created to actually create two
different methods one for runs the other for Clars.